tests/oeedger8r/behavior/deepcopy_oddsize.edl

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+enclave {
+  struct MyStruct {
+    [size=13] int* ptr;
```

Comment:
Can you add a comment why size is 13?
Is it intentionally not a multiple of sizeof(int)?

Reply:
Yup.

Reply:
I'm going to drop the scenario 6 yet because we don't actually enable its test (even in an ignored state).